code/modules/trade/datums/trade_stations_presets/zarya.dm

```diff
@@ -0,0 +1,40 @@
+/datum/trade_station/station_zarya
+	name_pool = list("FTS 'Zarya'" = "Free Trade Station 'Zarya', they sending message \"Privet, kosmicheskiy strannik, u nas mnogo engenernih nishtyakov, zaletai!\".")
```

Comment:
)))))))
Privet, you're speaking to the free station of Zarya. We have everything for sale here, don't be afraid to come aboard and check our wares!